src/logger.ts

```diff
@@ -13,9 +13,9 @@ const logger = createLogger({
     level: "info",
     format: combinedFormats,
     transports: [
```

Comment:
I would say that logger *must* handle creating the logs directory itself and not rely on the script running it create it

Reply:
agreed. Will do that here.